Add C standard library header.  John Dey, Fred Hutch Cancer Center 
diff -ruN ncbi-vdb-3.0.0.orig/vdb3/platform/memory/PrimordialMemoryMgr.cpp ncbi-vdb-3.0.0/vdb3/platform/memory/PrimordialMemoryMgr.cpp
--- ncbi-vdb-3.0.0.orig/vdb3/platform/memory/PrimordialMemoryMgr.cpp	2022-02-10 13:50:11.000000000 -0800
+++ ncbi-vdb-3.0.0/vdb3/platform/memory/PrimordialMemoryMgr.cpp	2022-08-18 14:24:21.242597000 -0700
@@ -24,6 +24,7 @@
 *
 */
 
+#include <cstdlib>
 #include <memory/PrimordialMemoryMgr.hpp>
 
 #include <memory/MemoryBlockItf.hpp>
